lib/rule_blocklist.js

```diff
@@ -0,0 +1,84 @@
+"use strict";
+
+const a = "a"
+  , Config = require("eslint/lib/config")
+  , merge = require("eslint/lib/config/config-ops").merge
+  ;
+
+const blocklistedRules = {
+  "no-unresolved": "Code Climate doesn't fetch your project's dependencies so this rule produces high amount of false positives."
+};
```

Comment:
Mentioned in another comment but repeating here inline: I don't think we need to support separate explanations for blocklisted rules just yet. This can be an array of rules and we can provide an explanation after updating the config.

`no-resolved` should be `import/no-resolved`.